pkg/synopsysctl/cmd_update.go

```diff
@@ -585,11 +585,11 @@ var updateBlackDuckAddEnvironCmd = &cobra.Command{
 	},
 }
 
-// updateBlackDuckAddRegistryCmd adds an Image Registry to a Black Duck instance
-var updateBlackDuckAddRegistryCmd = &cobra.Command{
-	Use:           "addregistry BLACK_DUCK_NAME REGISTRY",
-	Example:       "synopsysctl update blackduck addregistry <name> docker.io\nsynopsysctl update blackduck addregistry <name> docker.io -n <namespace>",
-	Short:         "Add an Image Registry to a Black Duck instance",
+// updateBlackDuckSetImageRegistryCmd adds an image to a Black Duck instance
+var updateBlackDuckSetImageRegistryCmd = &cobra.Command{
+	Use:           "setimage BLACK_DUCK_NAME IMAGE",
+	Example:       "synopsysctl update blackduck setimage <name> gcr.io/blackducksoftware/blackduck-cfssl:2019.6.0\nsynopsysctl update blackduck setimage <name> gcr.io/blackducksoftware/blackduck-cfssl:2019.6.0 -n <namespace>",
```

Comment:
I know it's just an example but those images are not valid. Can we replace gcr.io with docker.io?